larpix/format/message_format.py

```diff
@@ -73,6 +39,27 @@ def dataserver_message_encode(packets, key_parser=None, version=(1,0)):
          - byte[8:14] = 7-byte Unix timestamp
          - byte[15] is unused
 
+    A key parser should be provided to extract the ``'io_chain'`` from the packet chip key. If none is provided, io_chain
+    will be 0 for all packets. E.g.::
+
+        from larpix.larpix import Packet, Key
+        def ex_key_parser(key):
+            return dict(io_chain=key.io_channel)
+        packet = Packet()
+        packet.chip_key = Key('1-1-1')
+        msgs = datserver_message_encode([packet], key_parser=ex_key_parser)
+        msgs[0] # b'\\x01\\x00D\\x01\\x00\\x00\\x00\\x00\\x04\\x00\\x00\\x00\\x00\\x00\\x00\\x00'
```

Comment:
The bytestring here and below has extra backslashes in it (from repr maybe?)
```suggestion
        msgs[0] # b'\x01\x00D\x01\x00\x00\x00\x00\x04\x00\x00\x00\x00\x00\x00\x00'
```

Reply:
Rejected because of the idiosyncrasies of escape codes and bytes in docstrings - instead the docstring should begin with ``r'''`` to mean "raw," i.e. ignore escape sequences.